src/osd/pg_scrubber.cc

```diff
@@ -1865,6 +1876,23 @@ ostream& PgScrubber::show(ostream& out) const
   return out << " [ " << m_pg_id << ": " << m_flags << " ] ";
 }
 
+int PgScrubber::asok_debug(std::string_view cmd,
+			   std::string param,
+			   Formatter* f,
+			   stringstream& ss)
+{
+  dout(10) << __func__ << " cmd: " << cmd << " param: " << param << dendl;
+
+  if (cmd == "block") {
+    // set a flag that will cause the next 'select_range' to report a blocked object
+    m_debug_blockrange = 1;
+  } else if (cmd == "unblock") {
+    // send an 'unblock' event, as if a blocked range was freed
```

Comment:
What is the purpose of this?  Is it to let scrub continue?  If so, I can't think of any reason why this would be useful as it could produce incorrect scrub errors.

Reply:
This (the whole function - actually the whole 2'nd commit) is just a test rig.

Reply:
It cannot be reached during normal operation.

Reply:
@ronen-fr So are you adding a test using block/unblock to verify that the log message is generated?

Reply:
> So are you adding a test using block/unblock to verify that the log message is generated?

@dzafman plan to. Currently I am running this test manually. There are two issues that
deserve a followup PR: a formal test for this issue, and
continuing the change I made in osd.cc in order to support this test, i.e. the curving out
of identifting a target PG for an asok command etc (that change will require
more testing, thus I am separating it to a future PR)